serving/samples/helloworld-java-spark/README.md

```diff
@@ -0,0 +1,181 @@
+# Hello World - Spring Boot Java sample
+
+A simple web app written in Java using Spark Java Framework that you can use for
+testing.  
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed. Follow the
+  [installation instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- You have installed
+  [Java SE 8 or later JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html).
+
+## Recreating the sample code
+
+While you can clone all of the code from this directory, hello world apps are
+generally more useful if you build them step-by-step. The following instructions
+recreate the source files from this folder.
+
+1. From the console, unzip the helloworld.zip  using the unzip command:
+
+   ```shell
+   unzip helloworld.zip
```

Comment:
where is this zip coming from?

Reply:
ah it's a checked in binary.

Reply:
Why is there a checkin binary?
Can you instead ask the user to clone this repo?

Reply:
Added the step to clone the repo instead.